examples/google-force-safe-search/google-safe-search-domain-generator.rb

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env ruby
+
+# build google force safe search ruls for rubyDNS 
+
+google_domains = `curl https://www.google.com/supported_domains 2>> /dev/null | sed "s/^.//g"`.split("\n") 
```

Comment:
This can be improved.

```
require 'async'
require 'async/http/internet'

google_domains = Sync do
	internet = Async::HTTP::Internet.new
	response = internet.get("https://www.google.com/supported_domains")
	
	response.read.split(/\n/)
ensure
	response&.close
end
```